AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -3961,60 +3998,63 @@ function Debug-AzStorageAccountEntraKerbAuth {
             } catch {
                 $checks["CheckEntraObject"].Result = "Failed"
                 $checks["CheckEntraObject"].Issue = $_
-                Write-Error "CheckEntraObject - FAILED"
-                Write-Error $_
+                $aadTryCatchError = $_
+                Write-FailedPSStyle($aadTryCatchError)                
             }
         }
         #
         #Check if Reg key is enabled
         #
         if (!$filterIsPresent -or $Filter -match "CheckRegKey")
         {
+            [string] $aadObjIntro = "Checking Registry Key"
+            Write-Host $aadObjIntro
             try {
                 $checksExecuted += 1;
                 Write-Verbose "CheckRegKey - START"
-
+                
                 if (Test-IsCloudKerberosTicketRetrievalEnabled)
                 {
                     $checks["CheckRegKey"].Result = "Passed"
                     Write-Verbose "CheckRegKey - SUCCESS"
                 }
-                else {
+                else {                    
                     $checks["CheckRegKey"].Result = "Failed"
                     $checks["CheckRegKey"].Issue = "The CloudKerberosTicketRetrievalEnabled need to be enabled to get kerberos ticket"
-                    Write-Error "CheckRegKey - FAILED"
-                    Write-Error "The registry key HKLM\SYSTEM\CurrentControlSet\Control\Lsa\Kerberos\Parameters\CloudKerberosTicketRetrievalEnabled was non-existent or 0."
-                    Write-Error "For AAD Kerberos authentication, it should be set to 1."
-                    Write-Error "To fix this error, enable the registry key and reboot the machine."
-                    Write-Error "See https://learn.microsoft.com/en-us/azure/storage/files/storage-files-identity-auth-hybrid-identities-enable?tabs=azure-portal#configure-the-clients-to-retrieve-kerberos-tickets"
-                }
-                
+                    
+                    [string] $regKeyDoesNotExist = "The registry key $($PSStyle.Foreground.BrightCyan)HKLM\SYSTEM\CurrentControlSet\Control\Lsa\Kerberos\Parameters\CloudKerberosTicketRetrievalEnabled$($PSStyle.Reset) was non-existent or 0."

```

Comment:
Let's call this: the "CloudKerberosTicketRetrievalEnabled" setting was not set on this machine. To fix this error, see aka.ms/whatever"

Reply:
And let's shorten it